e2e/package.json

```diff
@@ -20,27 +20,27 @@
   "dependencies": {
     "@frequency-chain/api-augment": "file:../js/api-augment/dist/frequency-chain-api-augment-0.0.0.tgz",
     "@helia/unixfs": "^3.0.6",
-    "@polkadot/api": "11.2.1",
-    "@polkadot/types": "11.2.1",
+    "@polkadot/api": "12.0.1",
```

Comment:
Mentally noting this update, as it may force similar updates downstream in the gateway micro-services.

Reply:
We'll also want to see how this does with frequency-scenario-template. Note [this commit](https://github.com/AmplicaLabs/frequency-scenario-template/commit/9f7a22261bfd0aa79a97cf48dd2bca53fd6ac809) with the comment:
```
- [x] constrain `@frequency-chain/api-augment` package to `1.11.x`
versions until the upstream dependencies of >= `1.12.x` (namely
`@polkadot` packages >= 11.x) are better supported in our ecosystem
```

This affects the Gateway microservices downstream (it's a dev dependency for chain setup & e2e tests, but the package dependency tree causes issues with the app).

Would be nice to do some comprehensive testing & work out all of the downstream problems once and for all.